### ISSUE TITLE
При сворачивании приложения на выборе цвета - он слетает

### ISSUE THREAD

**Issue by OXOTH1K** [sprut/Home#35]
** Подробное описание ошибки **
Если при нахождении на экране выбора цвета свернуть приложение на телефонах с монобровью - выбирается нижний крайний цвет, который находится возле полоски для открытия многозадачности.

** Скриншоты/ Логи **

** Пожалуйста, заполните следующую информацию: **
  - Версия: IOS 14.1

**Comment by Batyushin** [comment]
Это ты его пальцем задеваешь. только что проверил раз 10. Если не касаться палитры, все хорошо

**Comment by OXOTH1K** [comment]
> Это ты его пальцем задеваешь. только что проверил раз 10. Если не касаться палитры, все хорошо

Согласен. Но только так никто не делает. Все свайпают по полосочке, а она близко к кругу с цветами)
